pages/auth/authorize.vue

```diff
@@ -68,6 +86,32 @@ import { useAuth } from '@/composables/auth.js'
 
 import { useScopes } from '@/composables/auth/scopes.ts'
 
+const { formatMessage } = useVIntl()
+
+const messages = defineMessages({
+  appInfo: {
+    id: 'auth.authorize.app-info',
+    defaultMessage:
+      '<strong>{appName}</strong> by <creator-link>{creatorUsername}</creator-link> will be able to:',
```

Comment:
```suggestion
      '<strong>{appName}</strong> by <creator-link>{creator}</creator-link> will be able to:',
```
I think ‘username’ suffix is unnecessary, either of those (`creator` or `username`) will be just fine.